src/patternfly/components/Table/table.scss

```diff
@@ -119,9 +119,6 @@
   // Standard table row (non-expandable)
   // exclude expandable rows
   tr:not(.pf-c-table__expandable-row) {
-    // set position relative for absolutely positioned elements in responsive view
```

Comment:
do you know why was this "necessary"?

Reply:
My guess is it was used previously but not anymore. There are no `position: absolute;` items that depend on the `tr` having `relative` positioning.